content/subscription/build-cloud/manage-seats.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Manage seats and invites
+description: Learn how to add and remove seats for your Docker Build Cloud subscription, and invite members to your team.
+keywords: subscription, team, business, features, build, cloud, Build Cloud, remote builder, manage users, add seats, remove seats
+---
+
+You can add seats and manage invitations to your Docker Build Cloud Team in the Docker Build Cloud dashboard.
+
+> **Note**
+>
+>If you have a Docker Build Cloud Business subscription, you can add and remove seats by working with your account executive, then assign your purchased seats in the Docker Build Cloud dashboard.
+
+## Add seats
+
+1. Sign in to your [Docker account](https://login.docker.com/) and go to [Docker Build Cloud](https://build.docker.com/).
```

Comment:
suggestion: Shouldn't it be "Go to Docker Build Cloud and sign in to your account"? Here and in the other step 1's. Do they need to take a detour via login.docker.com?

Reply:
It we do need the detour (signing in via a separate portal before going to the DBC dashboard, we should make it a separate step